### ISSUE TITLE
Icon request: icon-u-turn

### ISSUE THREAD

**Issue by tzin111** [FortAwesome/Font-Awesome#10023]
hi,
i need an u-turn icon.
i made some app with couple on langueges and i have LTR and RTL so the back button is a problem because its get flipped...
so, i need a universal back icon like u-turn icon. (i have it on my asus zen 2 android phone)

**Comment by tagliala** [comment]
![image](https://cloud.githubusercontent.com/assets/556268/19127202/54f36f50-8b3e-11e6-8a2e-4d0a69f92f76.png)

This icon?

**Comment by tzin111** [comment]
Yes but 90 deg to the right so it looks like back button. Here is camera
image from my tablet

בתאריך 5 באוק׳ 2016 21:58,‏ "Geremia Taglialatela" <notifications@github.com>
כתב:

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by jabronus** [comment]
These icons are now part of Font Awesome 7:

- https://fontawesome.com/icons/u-turn-down-left?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-down-right?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-left-down?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-left-up?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-right-down?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-right-up?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-up-left?f=classic&s=solid
- https://fontawesome.com/icons/u-turn-up-right?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-down-left?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-down-right?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-left-down?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-left-up?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-right-down?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-right-up?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-up-left?f=classic&s=solid
- https://fontawesome.com/icons/arrow-u-turn-up-right?f=classic&s=solid
